tests/unit/test_discovery_client.py

```diff
@@ -33,6 +34,9 @@
 from ni_measurementlink_service.measurement.info import MeasurementInfo, ServiceInfo
 from tests.utilities.fake_rpc_error import FakeRpcError
 
+if sys.platform.startswith("win"):
+    import win32file
```

Comment:
```
Run poetry run mypy tests
tests/unit/test_discovery_client.py:265: error: Name "win32file" is not defined  [name-defined]
Found 1 error in 1 file (checked 33 source files)
Error: Process completed with exit code 1.
```

```suggestion
if sys.platform == "win32":
    import win32file
```

Reply:
Actually, I tried both, and I can see in the pipeline that the tests are failing in both scenarios.

Reply:
nevermind, this part is fine